src/AdyenNativeModules.tsx

```diff
@@ -103,7 +123,7 @@ class AdyenNativeComponentWrapper implements AdyenActionComponent {
     this.nativeModule.open(paymentMethods, configuration);
   }
   hide(success: boolean, option?: { message?: string }) {
-    if (option != null && option.message != null) {
+    if (option && option.message) {
```

Comment:
We can simplify it to `if (option?.message)`

Reply:
Fixed [c7ec67c](https://github.com/Adyen/adyen-react-native/pull/331/commits/c7ec67ccfbc54ea22ba4184ff8f1cf37e508ef77)